CMakeLists.txt

```diff
@@ -5,14 +5,21 @@ find_package(ament_cmake REQUIRED)
 
 find_package(laser_geometry REQUIRED)
 find_package(message_filters REQUIRED)
+find_package(pcl_conversions REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(rclcpp_components REQUIRED)
 find_package(sensor_msgs REQUIRED)
 find_package(tf2 REQUIRED)
 find_package(tf2_ros REQUIRED)
 find_package(tf2_sensor_msgs REQUIRED)
+find_package(visualization_msgs REQUIRED)
```

Comment:
Will we use `ament_cmake_auto` and send a PR to upstream?

Reply:
Create PR on https://github.com/ros-perception/pointcloud_to_laserscan/pull/57

Reply:
Super cool! :smile:

Reply:
And could you please commit them to `tier4/main` and rebase this PR? :pray: 
Or will we wait for the response from upstream maintainers?

Reply:
I updated `tier4/main` and rebased this PR.